packages/odyssey-react-mui/src/Button.tsx

```diff
@@ -104,76 +104,83 @@ export type ButtonProps = {
 ) &
   SeleniumProps;
 
-const Button = ({
-  ariaDescribedBy,
-  ariaLabel,
-  ariaLabelledBy,
-  endIcon,
-  id,
-  isDisabled,
-  isFullWidth,
-  label = "",
-  onClick,
-  size = "medium",
-  startIcon,
-  testId,
-  tooltipText,
-  type = "button",
-  variant,
-}: ButtonProps) => {
-  const muiProps = useMuiProps();
-
-  const renderButton = useCallback(
-    (muiProps) => (
-      <MuiButton
-        {...muiProps}
-        aria-label={ariaLabel}
-        aria-labelledby={ariaLabelledBy}
-        aria-describedby={ariaDescribedBy}
-        data-se={testId}
-        disabled={isDisabled}
-        endIcon={endIcon}
-        fullWidth={isFullWidth}
-        id={id}
-        onClick={onClick}
-        size={size}
-        startIcon={startIcon}
-        type={type}
-        variant={variant}
-      >
-        {label}
-      </MuiButton>
-    ),
-    [
+const Button = forwardRef<HTMLElement, ButtonProps>(
+  (
+    {
```

Comment:
Is there a reason why `Button`, `Link`, and `Typography` don't also use `useImperativeHandle`?